core/src/test/scala/org/broadinstitute/dsde/rawls/metrics/RawlsStatsDTestUtils.scala

```diff
@@ -22,7 +22,7 @@ trait RawlsStatsDTestUtils extends StatsDTestUtils { this: Eventually with Mocki
   }
 
   protected def expectedWorkflowStatusMetric(workspace: Workspace, submission: Submission, workflowStatus: WorkflowStatus, expectedTimes: Int): (String, String) =
-    (s"${workbenchMetricBaseName}.workspace.${workspace.toWorkspaceName.toString.replace('/', '.')}.submission.${submission.submissionId}.workflowStatus.${workflowStatus.toString}.count", expectedTimes.toString)
+    (s"${workbenchMetricBaseName}.transient.workspace.${workspace.toWorkspaceName.toString.replace('/', '.')}.submission.${submission.submissionId}.workflowStatus.${workflowStatus.toString}.count", expectedTimes.toString)
```

Comment:
Covers a lot of tests, as all submission/workflow metric validation uses these methods.

Reply:
Heads up that I've refactored these methods and added a couple more in my PR.

Reply:
Yep, I'll update this